lib/commands/location.js

```diff
@@ -0,0 +1,37 @@
+import { exec } from 'teen_process';
+import { fs, util } from 'appium-support';
+import log from '../logger';
+
+let commands = {};
+
+commands.setGeoLocation = async function (location) {
+  let {latitude, longitude} = location;
+
+  if (!util.hasValue(latitude) || !util.hasValue(longitude)) {
+    log.errorAndThrow(`Both latitude and longitude should be set`);
+  }
+
+  if (!this.isSimulator()) {
+    if (await isIdeviceLocationAvailable()) {
```

Comment:
```
if (!await fs.which('idevicelocation')) {
  log.errorAndThrow(`idevicelocation binary should be present in PATH`);
}
```

There is no need for the separate `isIdeviceLocationAvailable` helper, since fs.which is not going to throw exceptions